packages/core/src/styles/global/index.scss

```diff
@@ -9,3 +9,6 @@
 @import "@faststore/ui/src/components/atoms/Overlay/styles.scss";
 @import "@faststore/ui/src/components/atoms/SROnly/styles.scss";
 @import "src/components/sections/Section/section.scss";
+
+// Importing this component because is being used outside the context of the Section
+@import "@faststore/ui/src/components/molecules/Dropdown/styles.scss";
```

Comment:
todo: I'll create another jira task so we can review this component implementation. (The "issue" here is that this Dropdown menu overlay is called outside the Breadcrumb section, so we need to import its style globally)

Reply:
follow-up task: https://vtex-dev.atlassian.net/browse/SFS-2308